examples/tts/radtts.py

```diff
@@ -59,6 +70,7 @@ def prepare_model_weights(model, unfreeze_modules):
 
 @hydra_runner(config_path="conf", config_name="rad-tts_dec")
 def main(cfg):
+    signal.signal(signal.SIGUSR1, handle_pdb)
```

Comment:
is it possible to register a signal interruption with a handler that triggers exception rather than debugger?

Reply:
There is no point to do that in signal handler - it won't be able to trigger exception properly

Reply:
Removed just in case.